nominees/community-health-toolkit.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Community Health Toolkit",
+  "aliases": [
+    "CHT"
```

Comment:
@nathanbaleeta Add Medic Mobile as an alias?

Reply:
Resolved in commit c246c15.